src/main/java/gov/cms/madie/models/cqm/datacriteria/EncounterPerformed.java

```diff
@@ -35,4 +35,5 @@ public class EncounterPerformed extends DataElement {
   private String qdmCategory = "encounter";
   private String qdmStatus = "performed";
   private String qdmVersion = "5.6";
+  private String _type = "QDM::EncounterPerformed";
```

Comment:
I see one issue in this class related to `private Code clazz`. Client is going to send `Code class` but because the class is reserved word in java we renamed it to `clazz`. we need to have a way to convert this back and forth otherwise we are going to loose it.

Reply:
new task is fine too but we need to note this so that we don't forget

Reply:
Ah I see what you mean, I originally modeled these after the ruby representations which does have `clazz` but the javascript does have `class` and a little conversion method for it. That's all a good point. I'll add a note in here in case we need something like that.